app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -2448,10 +2448,13 @@ class BrowserTabViewModel @Inject constructor(
 
     fun onUserClickCtaOkButton(cta: Cta) {
         ctaViewModel.onUserClickCtaOkButton(cta)
-        command.value = when (cta) {
+        val onboardingCommand = when (cta) {
```

Comment:
cc @nalcalag, i had to make this change to prevent a null command from being set